src/Uno.UI/UI/Xaml/Controls/TimePicker/TimeSpanExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace Windows.UI.Xaml.Controls
+{
+	internal static class TimeSpanExtensions
+	{
+		internal static TimeSpan RoundToInterval(this TimeSpan time, int interval)
```

Comment:
```suggestion
		internal static TimeSpan RoundToInterval(this TimeSpan time, int minuteInterval)
```
a way to clarify that the unit is minute.

Reply:
yes good point, i'll rename the method to RoundToNextMinuteInterval